src/DynamoCore/BuiltInAndOperators/BuiltIn.Migrations.xml

```diff
@@ -10,7 +10,13 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>ImportFromCSV</oldName>
-    <newName>DSCore.IO.Data.ImportCSV</newName>
+    <newName>DSOffice.Data.ImportCSV</newName>
+    <additionalAttributes>
+      <attribute>
+        <name>assembly</name>
+        <value>DSOffice.dll</value>
```

Comment:
Not entirely sure if this `assembly` is needed since it doesn't seem to have any effect on `BuiltIn` migrations